File: MeepMeepSequences/src/main/java/com/example/meepmeepsequences/MeepMeepSequences.java
package com.example.meepmeepsequences;

import com.example.meepmeepsequences.util.Context;
import com.example.meepmeepsequences.util.Detector;

public class MeepMeepSequences {
    public static void main(String[] args) {
        Context.location = Detector.Location.RIGHT;
        new BasicCarouselPath().carouselPath(false).start();
    }
}

File: MeepMeepSequences/src/main/java/com/example/meepmeepsequences/MeepMeepSequences.kt
package com.example.meepmeepsequences

import com.example.meepmeepsequences.util.Context
import com.example.meepmeepsequences.util.Context.windowSize
import com.example.meepmeepsequences.util.Detector
import com.example.meepmeepsequences.util.addMultiPath
import com.noahbres.meepmeep.MeepMeep
import com.noahbres.meepmeep.MeepMeep.Background

object MeepMeepSequences {
    @JvmStatic
    fun main(args: Array<String>) {
        Context.location = Detector.Location.RIGHT
        MeepMeep(windowSize)
            .setBackground(Background.FIELD_FREIGHTFRENZY_ADI_DARK)
            .setDarkMode(true)
            .setBackgroundAlpha(0.95f)
            .addMultiPath(BasicPaths()::carouselPath)
            .start()
    }
}